shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -811,6 +813,19 @@ object GlutenConfig {
       .intConf
       .createWithDefault(2)
 
+  val COLUMNAR_VELOX_SPILL_STRATEGY =
+    buildConf("spark.gluten.sql.columnar.backend.velox.spillStrategy")
+      .internal()
+      .doc(
+        "none: Disable spill on Velox backend; " +
+          "threshold: Use spark.gluten.sql.columnar.backend.velox.memoryCapRatio " +
+          "to calculate a memory threshold number for triggering spill; " +
+          "auto: Let Spark memory manager manage Velox's spilling")
+      .stringConf
+      .transform(_.toLowerCase(Locale.ROOT))
+      .checkValues(Set("none", "threshold", "auto"))
```

Comment:
I think these policies should be exclusive? e.g, if we set this to `auto`, the `threshold` based configurations shouldn't get any effects, is this correct?

Reply:
yes, the options are exclusive.